css-view-transitions-2/Overview.bs

```diff
@@ -577,7 +594,7 @@ When capturing the old or new state for an element, perform the following steps
 	</div>
 	<div algorithm>
 		To <dfn export>setup cross-document view-transition</dfn> given a {{Document}} |oldDocument|,
```

Comment:
Which task source is this algo called from? Can you leave a note.

Reply:
Why are task sources important here? It's an HTML concern